client/pages/users/users.js

```diff
@@ -15,4 +15,5 @@ export default function Profile(props) {
             <UserList />
         </>
     )
-}
\ No newline at end of file
+}
```

Comment:
@mwhitman189 get rid of props argument if it's not being used.
probably can use shorthand like this:

```
const funciton = () => (
   <Mycomponent ... />
)
```

Reply:
Thanks for the suggestion!

 I will remove props.
There are actually <Head> and <h1> elements, so I think the explicit return is necessary.